pkgs/development/python-modules/dronecan/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "dronecan";
+  version = "1.0.25";
+  disabled = pythonOlder "3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-0WKmVZwE6OgBckWWvPcn5BYqXMEt6Mr1P68UMHfRp4I=";
+  };
+
+  nativeCheckInputs = [
+    pytest
+  ];
+
+  # With the release of 1.0.23, the default check phase breaks, with the error
+  # TypeError: mcast.__init__() missing 1 required positional argument: 'url'
+  # So here I'm doing the tests the same way they're done upstream.
+  checkPhase = ''
+    runHook preCheck
+
+    python -m unittest discover test
+    python -m pytest
+
+    runHook postCheck
+  '';
```

Now the unittests are checked twice. `pytestCheckHook` helps here.

```suggestion
  nativeCheckInputs = [
    pytestCheckHook
  ];
```